automobile/ackermannvehicle.md

```diff
@@ -60,6 +62,8 @@ sensors, shape of the vehicle, etc.).
 - `boundingObject`: Physical geometry of the vehicle.
 - `physics`: `Physics` node of the vehicle defining the physical parameters of the
 vehicle.
+- `radarCrossSection`: Defines the `radarCrossSection` of the vehicle, if greater than 0, this vehicle is a potential target for any Radar device. It measures the vehicle ability to reflect radar signals in the direction of the radar receiver.
+- `recognitionColors`: Defines the `recognitionColors` of the vehicle. If not empty, this vehicle may be recognized by any Camera device with recognition capability.
```

Comment:
Great. Meanwhile, we can mention in this this new issue that those `radarCrossSection` and `recognitionColors` section could be hyperlinked to the reference manual, so that we can fix it once it work and leave it like that for now without hyperlink.